setup.py

```diff
@@ -97,29 +97,22 @@ def run(self):
 
 
 # Extra dependencies for remote integrations
+extra_requirements = {
+    path.stem: path.read_text().strip().splitlines()
+    for path in Path("requirements").glob("*.txt")
+}
```

Comment:
I am having a second thought here if we should move the requirements to `setup.cfg` and use pyup.io. 

That's what I use in webdav4: https://github.com/skshetry/webdav4/blob/main/setup.cfg.

Reply:
I also agree that is a better option, especially if we are migrating the regular metadata as well.